src/pybamm/dispatch/entry_points.py

```diff
@@ -1,6 +1,16 @@
+import hashlib
 import importlib.metadata
 import textwrap
+import urllib.request
 from collections.abc import Callable, Mapping
+from pathlib import Path
+
+from platformdirs import user_cache_dir
+
+from pybamm.expression_tree.operations.serialise import Serialise
+
+APP_NAME = "pybamm"
+APP_AUTHOR = "pybamm"
```

Comment:
I think we can keep just `APP_NAME` and drop `APP_AUTHOR`, and we can set `Path(user_cache_dir(APP_NAME)) / "models"` as a constant here instead.